rerun/src/test/java/org/apache/falcon/rerun/queue/InMemoryQueueTest.java

```diff
@@ -45,7 +45,7 @@ private void runTest() throws InterruptedException, FalconException {
             Thread.sleep(30);
             long time = System.currentTimeMillis();
             int delay = ((5 - index) / 2) * 50;
-            MyEvent event = new MyEvent("someCluster", Integer.toString(index),
+            MyEvent event = new MyEvent("someCluster", Integer.toString(index), "parent",
```

Comment:
Doesn't matter for this particular test as OozieWorkflowEngine API is not invoked.